app/models/court.rb

```diff
@@ -25,9 +25,24 @@ class Court < ActiveRecord::Base
     in: ['Outdoor without lights', 'Outdoor with lights', 'Indoor']
   }
 
+  validates_uniqueness_of :street_address,
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.